sys/kern/kasan.c

```diff
@@ -222,17 +236,12 @@ static void call_ctors(void) {
   }
 }
 
-static inline vaddr_t kasan_va_to_shadow(vaddr_t va) {
-  return KASAN_MD_SHADOW_START +
-         (va - KASAN_MD_SANITIZED_START) / KASAN_SHADOW_SCALE_SIZE;
-}
```

Comment:
It's not obvious why it chan be replaced by `addr_to_shad` if you look into that first time. Could you add a comment to commit message with explanation?

Reply:
Sure. I extended the description of the PR to explain the removal of `kasan_va_to_shadow`. I'll also tell @cahirwpz to mention it in the comment message while doing squash and merge.